### PR TITLE
[MIG][11.0]hr_emergency_contact:Migration to 11.0.

### PR DESCRIPTION
**Pull request by Trivedi-Vacha-SerpentCS**
[MIG][11.0]hr_emergency_contact:Migration to 11.0.

### CODE REVIEW

**Comment by hhgabelgaard** [comment]
Functionally tested
LGTM

File: hr_emergency_contact/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+Installation
+============
+
+To install this module, you need to:
+
+1.  Clone the branch 11.0 of the repository https://github.com/OCA/hr
+2.  Add the path to this repository in your configuration (addons-path)
+3.  Update the module list
+4.  Go to menu *Setting -> Modules -> Local Modules*
```
Comment (Swapnesh-SerpentCS) [inline hr_emergency_contact/README.rst]: *Apps -> Apps*

File: hr_emergency_contact/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+To use this module, you need to:
+
+1. Open employee data
+2. Go to *Emergency Contact* tab
+3. Add/select emergency contact
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/116/10.0
```
Comment (Swapnesh-SerpentCS) [inline hr_emergency_contact/README.rst]: Add correct URL for v11

**Review by nikul-serpentcs — APPROVED** [review]
Code Review LGTM

**Comment by nikul-serpentcs** [comment]
@pedrobaeza Ready to merge.

**Comment by pedrobaeza** [comment]
As always, please squash commit history, removing the merge operation, and grouping your 2 migration commits plus all the commits of andhit-r that were done on previous version for a clean commit history.

**Comment by nikul-serpentcs** [comment]
@pedrobaeza if it's Done then Move Forward

**Comment by LoisRForgeFlow** [comment]
@nikul-serpentcs Still not done, Pedro is referring to this commits: 
image

plus some reduction on the commits of andhit-r

**Comment by Trivedi-Vacha-SerpentCS** [comment]
@lreficent @pedrobaeza Squash done, Ready to Merge

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**